submit/submit

```diff
@@ -364,7 +377,8 @@ parser.add_argument('-c', '--contest', help='''submit for contest with ID or sho
 parser.add_argument('-p', '--problem', help='submit for problem with ID or label PROBLEM', default='')
 parser.add_argument('-l', '--language', help='submit in language with ID LANGUAGE', default='')
 parser.add_argument('-e', '--entry_point', help='set an explicit entry_point, e.g. the java main class')
-parser.add_argument('-v', '--verbose', help='increase verbosity', choices=loglevels.keys(), nargs='?', const='INFO', default='WARNING')
+parser.add_argument('-v', '--verbose',
+                    help='increase verbosity', choices=loglevels.keys(), nargs='?', const='INFO', default='WARNING')
```

Comment:
I'd argue that we should not break the line here: keeping it consistently on a single long line is better for readability than breaking it because it's longer than 120 characters.

Reply:
I'll check how easy it is to make CI ignore this